docker/Dockerfile

```diff
@@ -19,4 +19,4 @@ COPY --from=build /go/bin/hccore /usr/local/bin/hccore
 COPY --from=build /go/bin/hcd /usr/local/bin/hcd
 COPY --from=build /go/bin/hcdev /usr/local/bin/hcdev
 
-CMD ["/bin/bash"]
+CMD ["/usr/bin/env bash"]
```

Comment:
this breaks docker for me:

```
Davids-MacBook-Pro:docker davidmeister$ docker run -it d8acf419b886
docker: Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"/usr/bin/env bash\": stat /usr/bin/env bash: no such file or directory": unknown.
```

Does this change actually make sense for docker?

Reply:
Oh, it's complaining that `"/usr/bin/env bash"` isn't a file. Which is correct, since it's really an executable and an argument.

If the Docker image(s) can be guaranteed to have `/bin/bash`, then we can keep this line as `CMD ["/bin/bash"]`.

Reply:
well i'd expect us to have a lot more control over docker images, and if we change the image we can update `CMD` then too, i put up a related issue to smoke test docker builds https://github.com/holochain/holochain-proto/issues/672